### PR TITLE
added request_timeout to ollama, default set to 30.0 in settings.yaml…

### PR DESCRIPTION
**Pull request by sgresham**
… and settings-ollama.yaml

### CODE REVIEW

File: settings-ollama.yaml [hunk 1]
```diff
@@ -19,6 +19,7 @@ ollama:
   top_p: 0.9            # Works together with top-k. A higher value (e.g., 0.95) will lead to more diverse text, while a lower value (e.g., 0.5) will generate more focused and conservative text. (Default: 0.9)
   repeat_last_n: 64     # Sets how far back for the model to look back to prevent repetition. (Default: 64, 0 = disabled, -1 = num_ctx)
   repeat_penalty: 1.2   # Sets how strongly to penalize repetitions. A higher value (e.g., 1.5) will penalize repetitions more strongly, while a lower value (e.g., 0.9) will be more lenient. (Default: 1.1)
+  request_timeout: 30.0 # Time elapsed until ollama times out the request. Default is 30s. Format is float.
```
Comment (imartinez) [inline settings-ollama.yaml:22]: Could you default it to 60 or even 90? People running on CPU are having trouble with 30.

**Review by imartinez — COMMENTED** [review]
Love it, thanks! Added a comment, let me know what you think

**Review by imartinez — CHANGES_REQUESTED** [review]
Aren't you missing the addition of the setting to `settings.py`? 
I'd add it with a default value there. Otherwise passing a None value may break the integration.

**Comment by sgresham** [comment]
I have added settings now to settings.py. Apologies for missing it.

**Review by imartinez — APPROVED** [review]

**Comment by imartinez** [comment]
Merged in https://github.com/zylon-ai/private-gpt/pull/1773

**Pull request closed without merging**